source/opt/eliminate_dead_output_stores_pass.cpp

```diff
@@ -45,6 +47,64 @@ bool EliminateDeadOutputStoresPass::IsLiveBuiltin(uint32_t bi) {
   return live_builtins_->find(bi) != live_builtins_->end();
 }
 
+bool EliminateDeadOutputStoresPass::IsVariableRead(
+    Instruction const* const variable) {
+  analysis::DefUseManager* def_use_mgr = context()->get_def_use_mgr();
+
+  bool is_read = false;
+  def_use_mgr->ForEachUser(variable, [&is_read](Instruction* user) {
```

Comment:
Does this need to iterate over every user? If not, should `WhileEachUser` be used instead?